### PR TITLE
Add single binary deployment support

### PR DESCRIPTION
**Pull request by mvgorbunov**
[Improvement] Add single binaries deployment support, instead of archive.

### CODE REVIEW

File: roles/ydbd/tasks/main.yaml [hunk 1]
```diff
@@ -21,10 +21,17 @@
     update_cache: "{{ apt_update_cache | default(True) }}"
   timeout: 300
 
-- name: install custom libidn from archive
```
Comment (blinkov) [inline roles/ydbd/tasks/main.yaml:24]: Was this libidn thing intentionally dropped? It looks unrelated to subj.

File: roles/ydbd/tasks/main.yaml [hunk 2]
```diff
@@ -21,10 +21,17 @@
...
+- name: install YDB server and cli binaries
+  block:
+  - name: install YDB server binary package from archive
+    import_tasks:
+      file: install_ydb_from_archive.yaml
+    when: ydb_archive is defined
+
+  - name: install YDB binaries from files
+    import_tasks:
+      file: install_ydb.yaml
```
Comment (artgromov) [inline roles/ydbd/tasks/main.yaml:33]: Suggest: rename to `install_ydb_using_binary.yaml`. `install_ydb` is too generic.

File: roles/ydbd/defaults/main.yaml [hunk 3]
```diff
@@ -2,5 +2,6 @@ ydb_libidn_archive_unpack_options: ['--strip-component=1']
 
 ydb_archive_unpack_options: ['--strip-component=1']
 ydb_archive_force_install: false
+ydb_force_install: false
```
Comment (artgromov) [inline roles/ydbd/defaults/main.yaml:5]: Suggest: rename to `ydb_binary_force_install`. `ydb_force_install` seems too generic.

File: roles/ydbd/tasks/main.yaml (hunk 2 above)
Comment (mvgorbunov) [reply roles/ydbd/tasks/main.yaml]: Fixed

File: roles/ydbd/tasks/main.yaml (hunk 1 above)
Comment (mvgorbunov) [reply roles/ydbd/tasks/main.yaml:24]: Fixed, thanks

**Comment by mvgorbunov** [comment]
The main idea is that you can initially setup your cluster from tar.gz archive (include ydbd, ydb-cli, *.so) and then you can easily update ydbd and/or ydb cli binary with your own (e.g. local build from main)

**Review by artgromov — APPROVED** [review]
